tests/postgres/tdbtypes.nim

```diff
@@ -33,7 +32,7 @@ suite "Import dbTypes from norm/private/postgres/dbtypes":
 
   test "dbValue[DateTime] is imported":
     let users = @[newUser()].dup:
-      dbConn.select("""lastLogin <= $1""", ?now())
+      dbConn.select(""""lastLogin" <= $1""", ?now())
```

Comment:
An example for suddenly required quotation marks, this time not on an FK field as lastLogin is of type DateTime. Not sure what that is about.